user-operator/src/test/java/io/strimzi/operator/user/model/acl/SimpleAclRuleTest.java

```diff
@@ -46,11 +50,30 @@ public void testFromCrd()   {
             .withOperation(AclOperation.READ)
             .build();
 
-        SimpleAclRule simple = SimpleAclRule.fromCrd(rule);
-        assertThat(simple.getOperation(), is(AclOperation.READ));
-        assertThat(simple.getType(), is(AclRuleType.ALLOW));
-        assertThat(simple.getHost(), is("127.0.0.1"));
-        assertThat(simple.getResource(), is(resource));
+        List<SimpleAclRule> simpleAclRules = SimpleAclRule.fromCrd(rule);
+        assertThat(simpleAclRules.get(0).getOperation(), is(AclOperation.READ));
+        assertThat(simpleAclRules.get(0).getType(), is(AclRuleType.ALLOW));
+        assertThat(simpleAclRules.get(0).getHost(), is("127.0.0.1"));
+        assertThat(simpleAclRules.get(0).getResource(), is(resource));
+    }
+
+    @Test
+    public void testFromCrdWithThreeOperationsPerRule()   {
```

Comment:
Are there some tests which still test the old layout with single operation?

Reply:
I think the test above does it.